Cargo.toml

```diff
@@ -118,14 +126,30 @@ name = "fish_key_reader"
 path = "src/bin/fish_key_reader.rs"
 
 [features]
-default = ["embed-data"]
+default = ["embed-data", "localize-all-languages"]
 benchmark = []
 embed-data = ["dep:rust-embed", "dep:fish-build-man-pages"]
 # This feature is used to enable extracting messages from the source code for localization.
 # It only needs to be enabled if updating these messages (and the corresponding PO files) is
 # desired. This happens when running tests via `build_tools/check.sh` and when calling
 # `build_tools/update_translations.fish`, so there should not be a need to enable it manually.
 gettext-extract = ["dep:fish-gettext-extraction"]
+# These features control for which languages gettext localizations are built.
+# Not enabling localizations (or only enabling them for certain languages) reduces binary sizes, and
+# reduces the work required on compilation.
+# These features are used in `build.rs` and `src/wutil/gettext.rs`.
+localize-all-languages = ["localize", "fish-gettext-maps/all-languages"]
```

Comment:
I wonder if there's a better name for this feature flag.
Maybe "localize" or "localize-all" make more sense.. but I'm not sure, no need to change it.